service/src/tests/commons.rs

```diff
@@ -44,106 +38,11 @@ pub struct Message {
 	pub sha256: sgx_sha256_hash_t,
 }
 
-/// Who must be root account
-pub fn encrypted_set_balance<E: EnclaveBase>(
-	enclave_api: &E,
-	who: AccountKeyring,
-	nonce: Index,
-) -> Vec<u8> {
-	info!("*** Get the public key from the TEE\n");
-	let rsa_pubkey: Rsa3072PubKey = enclave_api.get_rsa_shielding_pubkey().unwrap();
-	info!("deserialized rsa key");
-
-	let call = TrustedCall::balance_set_balance(who.public().into(), who.public().into(), 33, 44);
-	encrypt_payload(
-		rsa_pubkey,
-		call.sign(
-			&KeyPair::Sr25519(who.pair()),
-			nonce,
-			&enclave_api.get_mrenclave().unwrap(),
-			&ShardIdentifier::default(),
-		)
-		.encode(),
-	)
-}
```

Comment:
So the functions here were only used in the integration tests, that's why they're removed?

Reply:
Yes that were my thoughts.